packages/ove-asset-manager/src/OVE.Service.AssetManager/Migrations/20181009081536_InitialCreate.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;

```

Comment:
Is this class auto-generated? If not why `ReSharper disable All`?

Reply:
yes migrations are autogenerated

Reply:
I just realised that not all classes have the comment that says auto-generated.